Plugins/Public/alley/shipclass_item.cpp

```diff
@@ -279,6 +279,21 @@ void SCI::CheckItems(unsigned int iClientID)
 	return;
 }
 
+bool SCI::CanBuyItem(uint iArchID, uint iClientID) {
+
+	auto foundItem = shipclassitems.find(iArchID);
+	if (foundItem != shipclassitems.end()) {
+		Archetype::Ship* shipArch = Archetype::GetShip(Players[iClientID].iShipArchetype);
+		for (uint allowedClass : foundItem->second.canmount) {
```

Comment:
just wondering if `canmount` is appropriate for purchasing restrictions
and on that subject don't we also need to be looking at `nomount`?